src/libs/actions/Report.ts

```diff
@@ -3644,7 +3647,9 @@ function prepareOnboardingOptimisticData(
         type: 'task',
         task: task.type,
         taskReportID: currentTask.reportID,
-        parentReportID: currentTask.parentReportID ?? '-1',
+        // API expects a string, that's why policyID must default to a string
+        // eslint-disable-next-line rulesdir/no-default-id-values
+        parentReportID: currentTask.parentReportID ?? '',
```

Comment:
> API expects a string

Are you sure? How was it before TS migration?  Is it be possible to call this API call without `parentReportID` or with invalid `parentReportID`?

Reply:
@VickyStash I'm not sure what the API expects or how to test it, but currently we send '-1' in case `currentTask.parentReportID` is `undefined`

Reply:
Answered here: https://github.com/Expensify/App/pull/54306#discussion_r1893603705